www/includes/easyparliament/templates/html/people/index.php

```diff
@@ -0,0 +1,235 @@
+                        <span class="people-list__person__party <?= strtolower( $mp_data['party'] ) ?>"><?= $mp_data['party'] ?></span>
+                        </p>
+                    </a>
+                    <?php }
+                    if ( isset($reps) ) {
+                        if ( isset($mp_data) && $type != 'mlas' ) { ?>
+                    <div class="people-list__your-mp__replist-header">
+                        <?php } ?>
+                      <h3>Your <?= $rep_plural == 'MSPs' ? 'regional ' : '' ?><?= $rep_plural ?> are</h3>
+                    </div>
+                        <?php foreach ( $reps as $rep ) { ?>
+                    <a href="<?= $rep['mp_url'] ?>" class="people-list__person">
+                    <img class="people-list__person__image" src="<?= $rep['image'] ?>">
+                        <h2 class="people-list__person__name"><?= $rep['name'] ?></h2>
+                        <p class="people-list__person__memberships">
+                        <span class="people-list__person__constituency"><?= $rep['constituency'] ?></span>
+                        <span class="people-list__person__party <?= strtolower( $rep['party'] ) ?>"><?= $rep['party'] ?></span>
+                        </p>
+                    </a>
+                        <?php } ?>
+                    <?php } ?>
+                </div>
+            </div>
+        </div>
+
+      <?php } else { ?>
+
+        <form action="/search/">
+            <div class="search-page__section search-page__section--search">
+                <div class="search-page__section__primary">
+                    <p class="search-page-main-inputs">
+                    <?php if ( $type == 'peers' ) { ?>
+                        <label for="find-mp-by-name-or-postcode">Find <?= $rep_plural ?> by name:</label>
+                    <?php } else { ?>
+                        <label for="find-mp-by-name-or-postcode">Find your <?= $rep_name ?> by name or postcode:</label>
+                    <?php } ?>
+                        <input type="text" class="form-control" name="q" id="find-mp-by-name-or-postcode">
+                        <button type="submit" class="button">Find</button>
+                    </p>
+                </div>
+            </div>
+        </form>
+
+      <?php } ?>
+
+        <div class="search-page__section search-page__section--results">
+        <?php if ($type == 'mps' && !count($data) && DISSOLUTION_DATE) {
+            # No MPs. Election period!
+        ?>
+            <div class="search-page__section__primary">
+                During the period from the dissolution of Parliament to the general election, there are no Members of Parliament.
+                <a href="/mps/?date=<?=DISSOLUTION_DATE ?>">View list of MPs as it was when Parliament was dissolved</a>
+            </div>
+        <?php } elseif ($type == 'msps' && !count($data) && DISSOLUTION_DATE) { ?>
```

Comment:
Afraid you don't want the `DISSOLUTION_DATE` here, it's an MP-only thing at the mo, I think.
Other than that, woo :)